src/modules/Dashboard/components/DashboardTable/DashboardTable.styles.ts

```diff
@@ -19,8 +23,11 @@ export const StyledTableCell = styled(TableCell, shouldForwardProp)`
       `;
     }
 
-    return `
-      height: 4.8rem;
-    `;
+    return { paddingTop: 12, paddingBottom: 12 };
```

Comment:
Please use theme.spacing for paddings and margins. Please also check other occurrences.